app/src/main/java/pl/szczodrzynski/edziennik/data/db/entity/Teacher.kt

```diff
@@ -83,6 +81,9 @@ open class Teacher {
     @ColumnInfo(name = "teacherId")
     var id: Long
 
+    @ColumnInfo(name = "teacherGlobalKey")
+    var globalKey: String? = null
+
```

Comment:
This is what `loginId` is for - this is a freeform string that can store register-dependent ID data. The `LoginId` from previous API is probably not needed now, so it can store the global key instead. Create another endpoint to receive the messagebox addressbook and store the global keys in `loginId`.